raiden/storage/serialization/fields.py

```diff
@@ -11,24 +11,14 @@
 from raiden.utils.typing import Address, Any, ChainID, ChannelID, Optional, Tuple
 
 
-class IntegerToStringField(marshmallow.fields.Field):
-    def _serialize(self, value: int, attr: Any, obj: Any, **kwargs: Any) -> str:
-        return str(value)
+class IntegerToStringField(marshmallow.fields.Integer):
```

Comment:
Did you find out why/how this gets used in the services?

Reply:
Yes, raiden monkeypatches `_native_to_marshmallow` to provide fields for the types defined in raiden in `raiden/storage/serialization/types.py`. This way the `IntegerToStringField` gets used for e.g. the `BlockNumber` typed `expiration_block` field. This is hackish due to the monkeypatching but useful, since it allows us to use raiden types in a much more concise and less repetitive way.